samples/README_Windows.md

```diff
@@ -39,7 +39,7 @@ https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt
 As an example, if the Open Enclave SDK is installed to `C:\openenclave`, then you would set `CMAKE_PREFIX_PATH` as shown below
 
 ```cmd
-set CMAKE_PREFIX_PATH=C:\openenclave\lib\openenclave\cmake
+set CMAKE_PREFIX_PATH=C:\oe\open-enclave\openenclave\lib\openenclave\cmake
```

Comment:
Does this match all the samples? https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md#buildinginstallation-on-windows-using-developer-command-prompt for example, we set the install path as c:\openenclave. And that document follows into this. Can we have the same path everywhere?

Reply:
All the docs ought to match.  It looks like this PR updates some places and not other places, making the docs inconsistent.  For example, it looks like the following docs still use `C:\openenclave`:
* GettingStartedDocs\Windows_vscode.md
* GettingStartedDocs\Contributors\WindowsInstallInfo.md
* GettingStartedDocs\Contributors\building_oe_sdk.md
* GettingStartedDocs\Contributors\WindowsSGX1FLCGettingStarted.md
* GettingStartedDocs\Contributors\WindowsSGX1GettingStarted.md

I think all the docs should be kept consistent one way or the other before this is merged.

Reply:
@dthaler GettingStartedDocs\Windows_vscode.md uses the path not only in text but also in screenshots. I'd prefer open another follow-up issue for updating the this specific documentation (i.e., requiring someone tests the vscode debugging while updating). What do you think?

Reply:
I think it's easier to standardize on c:\openenclave and not c:\oe in all examples, since most docs already use the former.   Then I don't think you need to modify the other files, just revert some changes in this PR.  What do you think?

Reply:
Sure. I think the only inconsistency part is really the instruction on the installation via the nuget package ([doc](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Windows.md#download-and-install-the-open-enclave-sdk-nuget-package)).

Following this instruction, the openenclave is installed to the C:\oe\open-enclave\openenclave rather than C:\openenclave, which is currently the only place uses this path among the documentation.

The easy fix would be just adding an instruction/comment that resolves/explains the difference here. Does it sound good to you?

Reply:
I didn't follow, but if there's a difference then sure go ahead and propose text and maybe I'll understand then :)   Thanks for looking into the consistency issue.

Reply:
Please check the updated version. Thanks!